from PyQt6 import QtCore, QtGui, QtWidgets


class Ui_SingUpWindow(object):
    def setupUi(self, SingUp):
        SingUp.setObjectName("SingUp")
        SingUp.resize(500, 500)
        font = QtGui.QFont()
        font.setFamily("Papyrus")
        font.setPointSize(12)
        SingUp.setFont(font)
        SingUp.setStyleSheet("background-color: rgb(24, 46, 17);")
        self.logoBadge = QtWidgets.QLabel(parent=SingUp)
        self.logoBadge.setGeometry(QtCore.QRect(70, 80, 61, 51))
        self.logoBadge.setStyleSheet("image: url(../Resources/logo.png);")
        self.logoBadge.setText("")
        self.logoBadge.setObjectName("logoBadge")
        self.namingLabel = QtWidgets.QLabel(parent=SingUp)
        self.namingLabel.setGeometry(QtCore.QRect(140, 80, 281, 61))
        font = QtGui.QFont()
        font.setFamily("Papyrus")
        font.setPointSize(50)
        font.setBold(False)
        font.setItalic(False)
        font.setStrikeOut(False)
        font.setKerning(True)
        self.namingLabel.setFont(font)
        self.namingLabel.setObjectName("namingLabel")
        self.nameTextEdit = QtWidgets.QLineEdit(parent=SingUp)
        self.nameTextEdit.setGeometry(QtCore.QRect(100, 190, 300, 41))
        font = QtGui.QFont()
        font.setFamily("Papyrus")
        font.setPointSize(18)
        self.nameTextEdit.setFont(font)
        self.nameTextEdit.setLayoutDirection(QtCore.Qt.LayoutDirection.LeftToRight)
        self.nameTextEdit.setStyleSheet("background-color: rgb(43, 54, 44);")
        self.nameTextEdit.setText("")
        self.nameTextEdit.setFrame(True)
        self.nameTextEdit.setEchoMode(QtWidgets.QLineEdit.EchoMode.Normal)
        self.nameTextEdit.setDragEnabled(False)
        self.nameTextEdit.setClearButtonEnabled(True)
        self.nameTextEdit.setObjectName("nameTextEdit")
        self.passwordTextEdit1 = QtWidgets.QLineEdit(parent=SingUp)
        self.passwordTextEdit1.setGeometry(QtCore.QRect(100, 310, 300, 41))
        font = QtGui.QFont()
        font.setFamily("Papyrus")
        font.setPointSize(18)
        self.passwordTextEdit1.setFont(font)
        self.passwordTextEdit1.setLayoutDirection(QtCore.Qt.LayoutDirection.LeftToRight)
        self.passwordTextEdit1.setStyleSheet("background-color: rgb(43, 54, 44);")
        self.passwordTextEdit1.setEchoMode(QtWidgets.QLineEdit.EchoMode.Password)
        self.passwordTextEdit1.setClearButtonEnabled(True)
        self.passwordTextEdit1.setObjectName("passwordTextEdit1")
        self.signupButton = QtWidgets.QPushButton(parent=SingUp)
        self.signupButton.setGeometry(QtCore.QRect(180, 440, 141, 41))
        font = QtGui.QFont()
        font.setFamily("Papyrus")
        font.setPointSize(24)
        font.setBold(True)
        font.setItalic(False)
        self.signupButton.setFont(font)
        self.signupButton.setStyleSheet("background-color: rgb(51, 104, 36);")
        self.signupButton.setAutoDefault(True)
        self.signupButton.setDefault(False)
        self.signupButton.setFlat(False)
        self.signupButton.setObjectName("signupButton")
        self.passwordTextEdit2 = QtWidgets.QLineEdit(parent=SingUp)
        self.passwordTextEdit2.setGeometry(QtCore.QRect(100, 360, 300, 41))
        font = QtGui.QFont()
        font.setFamily("Papyrus")
        font.setPointSize(18)
        self.passwordTextEdit2.setFont(font)
        self.passwordTextEdit2.setLayoutDirection(QtCore.Qt.LayoutDirection.LeftToRight)
        self.passwordTextEdit2.setStyleSheet("background-color: rgb(43, 54, 44);")
        self.passwordTextEdit2.setEchoMode(QtWidgets.QLineEdit.EchoMode.Password)
        self.passwordTextEdit2.setClearButtonEnabled(True)
        self.passwordTextEdit2.setObjectName("passwordTextEdit2")
        self.emailTextEdit = QtWidgets.QLineEdit(parent=SingUp)
        self.emailTextEdit.setGeometry(QtCore.QRect(100, 240, 300, 41))
        font = QtGui.QFont()
        font.setFamily("Papyrus")
        font.setPointSize(18)
        self.emailTextEdit.setFont(font)
        self.emailTextEdit.setLayoutDirection(QtCore.Qt.LayoutDirection.LeftToRight)
        self.emailTextEdit.setStyleSheet("background-color: rgb(43, 54, 44);")
        self.emailTextEdit.setText("")
        self.emailTextEdit.setFrame(True)
        self.emailTextEdit.setEchoMode(QtWidgets.QLineEdit.EchoMode.Normal)
        self.emailTextEdit.setDragEnabled(False)
        self.emailTextEdit.setClearButtonEnabled(True)
        self.emailTextEdit.setObjectName("emailTextEdit")
        self.backButton = QtWidgets.QPushButton(parent=SingUp)
        self.backButton.setGeometry(QtCore.QRect(10, 20, 71, 21))
        font = QtGui.QFont()
        font.setFamily("Papyrus")
        font.setPointSize(14)
        font.setBold(True)
        font.setItalic(False)
        self.backButton.setFont(font)
        self.backButton.setStyleSheet("background-color: rgb(36, 74, 27);")
        self.backButton.setAutoDefault(True)
        self.backButton.setDefault(False)
        self.backButton.setFlat(False)
        self.backButton.setObjectName("backButton")

        self.retranslateUi(SingUp)
        QtCore.QMetaObject.connectSlotsByName(SingUp)

    def retranslateUi(self, SingUp):
        _translate = QtCore.QCoreApplication.translate
        SingUp.setWindowTitle(_translate("SingUp", "Sing Up Menu"))
        self.namingLabel.setText(_translate("SingUp", "Let\'s recycle"))
        self.nameTextEdit.setPlaceholderText(_translate("SingUp", "name"))
        self.passwordTextEdit1.setPlaceholderText(_translate("SingUp", "password"))
        self.signupButton.setText(_translate("SingUp", "Sign up"))
        self.passwordTextEdit2.setPlaceholderText(_translate("SingUp", "repeat password"))
        self.emailTextEdit.setPlaceholderText(_translate("SingUp", "email"))
        self.backButton.setText(_translate("SingUp", "<-back"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    SingUp = QtWidgets.QDialog()
    ui = Ui_SingUpWindow()
    ui.setupUi(SingUp)
    SingUp.show()
    sys.exit(app.exec())
